src/core/engines/Cesium/Feature/index.tsx

```diff
@@ -70,56 +73,85 @@ const pickProperty = (k: keyof AppearanceTypes, layer: ComputedLayer) => {
   return layer.layer[k];
 };
 
+const CACHED_COMPONENTS = new LRUCache<string, JSX.Element>({ max: 10000 });
```

Comment:
I think this LRUCache is instantiated by each layer. Doesn't this press memory?

Reply:
It does but the memory signature isn't that significant. If you want to we can reduce the maxSize.